libs/domain/search/src/services/revealers/suggestion-experience-data.revealer.spec.ts

```diff
@@ -3,7 +3,7 @@ import { createInjector, destroyInjector, getInjector } from '@spryker-oryx/di';
 import { MessageType, postMessage } from '@spryker-oryx/experience';
 import { RouteType } from '@spryker-oryx/router';
 import { of } from 'rxjs';
-import { SuggestionField } from '../adapter';
+import { SuggestionField } from '../adapter/spryker-glue';
```

Comment:
Is there a way to define the path for all of these imparts? all at once?
Typically you will have one set of adapters used everywhere. Cases where I need to specify a separate import directory when multiple vendors involved are limited, and then it can be a little more complex to do. But per default it should be a one-time switcher.